Configuration/Eras/python/Modifier_pbpb_run3_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+pbpb_run3 =  cms.Modifier()
```

Comment:
this is not needed anymore.
If a placeholder is desired, a name following existing pattern would be pp_on_AA_run3 or pp_on_PbPb_run3 if finer specification is desired